src/main/kotlin/com/hibob/academy/entity/types.kt

```diff
@@ -42,7 +51,11 @@ enum class PetType {
 
     companion object {
         fun fromString(type: String): PetType {
-            return valueOf(type.uppercase())
+            return try {
+                valueOf(type.uppercase())
+            } catch (e: IllegalArgumentException) {
```

Comment:
This is not related to this PR
We try to focus on one change area - here it is the generated id